projects/Mallard/src/helpers/analytics/index.ts

```diff
@@ -0,0 +1,52 @@
+import {
+	Action,
+	ComponentType,
+	sendComponentEvent,
+	sendPageViewEvent,
+	setUserId,
+} from 'src/services/ophan';
+import type { AnalyticsEvent, AnalyticsUserId } from './types';
+
+// Currently not used in Ophan as this refers to user behaviour rather than journalism
```

Comment:
Do we need to remove these comments?

Reply:
Yep as its there to define the API. Next PR, this get uncommented and used.